content/kubermatic/main/architecture/compatibility/supported-versions/_index.en.md

```diff
@@ -31,22 +31,19 @@ these migrations.
 In the following table you can find the supported Kubernetes versions for the
 current KKP version.
 
-| KKP version  | 1.25     | 1.24[^1] | 1.23[^1] | 1.22[^2] | 1.21[^2] | 1.20[^2] | 1.19[^2]   |
-| ------------ | -------- | -------- | -------- | -------- | -------- | -------- | ---------- |
-| 2.22.x[^not-out-yet] | ✓        | ✓        | ✓        | -        | -        | -        | -          |
-| 2.21.x       | -        | ✓        | ✓        | ✓        | ✓        | -        | -          |
-| 2.20.x       | -        | -        | ✓[^4]    | ✓        | ✓        | ✓        | -          |
-| 2.19.x       | -        | -        | -        | ✓        | ✓        | ✓        | -          |
-| 2.18.x[^3]   | -        | -        | -        | ✓        | ✓        | ✓        | ✓          |
+| KKP version          | 1.26 | 1.25 |1.24 | 1.23[^2]| 1.22[^2] | 1.21[^2] | 1.20[^2] | 1.19[^2] |
+| -------------------  | ---- | ---- | --- | ------- | -------- | -------- | -------- | -------- |
+| 2.22.x[^not-out-yet] | ✓    | ✓    | ✓   | -       | -        | -        | -        | -        |
+| 2.21.x               | -    | -    | ✓   | ✓       | ✓        | -        | -        | -        |
+| 2.20.x               | -    | -    | -   | ✓       | ✓        | ✓        | ✓        | -        |
+| 2.19.x               | -    | -    | -   | -       | ✓        | ✓        | ✓        | -        |
+| 2.18.x[^3]           | -    | -    | -   | -       | ✓        | ✓        | ✓        | ✓        |
 
-[^1]: Kubernetes 1.24 and 1.23 are currently not supported on ARM64 clusters with Canal CNI and kube-proxy running in the IPVS mode.
 
-[^2]: Kubernetes 1.19, 1.20, 1.21 and 1.22 releases have reached End-of-Life (EOL). We strongly recommend upgrading to a supported Kubernetes release as soon as possible.
+[^2]: Kubernetes releases from 1.19 to 1.23 have reached End-of-Life (EOL). We strongly recommend upgrading to a supported Kubernetes release as soon as possible.
```

Comment:
Idea: showing only N, N-1 and N-2 KKP versions in each version's support matrix. This way we can pretty much avoid the need of these kind of comments.

Here that would mean showing only 2.22, 2.21 and 2.20, because they are supported currently, when this version of the documentation will be in effect.

Reply:
Good call, I've removed 2.19 and 2.18 from the list since both will be unsupported with the 2.22 release.